autoload/lsp/utils/diff.vim

```diff
@@ -0,0 +1,115 @@
+" This is copied from https://github.com/natebosch/vim-lsc/blob/master/autoload/lsc/diff.vim
```

Comment:
We do need to explicitly refer their license too. For example: https://github.com/rust-lang/cargo/blob/master/LICENSE-THIRD-PARTY